sw/device/tests/dif/dif_clkmgr_unittest.cc

```diff
@@ -0,0 +1,293 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/dif/dif_clkmgr.h"
+
+#include "gtest/gtest.h"
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/testing/mock_mmio.h"
+
+#include "clkmgr_regs.h"
+
+namespace dif_clkmgr_unittest {
+namespace {
+using mock_mmio::MmioTest;
+using mock_mmio::MockDevice;
+using testing::Test;
+
+class ClkMgrTest : public Test, public MmioTest {
+ protected:
+  const dif_clkmgr_gateable_clock_t default_last_gateable_clock = 4;
+  const dif_clkmgr_hintable_clock_t default_last_hintable_clock = 6;
```

Comment:
If these are intended to be constants, they should be `static const` and have a name like `kMyConst`.